fv3gfs-physics/tests/savepoint/translate/translate_fv_update_phys.py

```diff
@@ -34,9 +33,11 @@ def __getitem__(self, item):
 
 
 class TranslateFVUpdatePhys(ParallelTranslate2Py):
-    def __init__(self, grids):
-        super().__init__(grids)
+    def __init__(self, grids, namelist, stencil_factory):
+        super().__init__(grids, namelist, stencil_factory)
```

Comment:
Technically it shouldn't be necessary to pass `stencil_factory` separate from `grids`, since each `grid` object has its own stencil factory. It would be different if we are able to remove `grid`, it also would be different if we could remove the `Grid.stencil_factory` attribute and _only_ make stencil factory directly.

Do you think for now we should remove the stencil_factory argument? I can see keeping it if you think this is a step on a road towards retrieving fewer attributes from Grid and eventually being able to use it only for array transformations. I'm kind of undecided on this one.

Reply:
Yes, let's remove the `stencil_factory` argument and access it through grid.